### ISSUE TITLE
socket io on background

### ISSUE THREAD

**Issue by wepashka** [rikulo/socket.io-client-dart#307]
How can I use socket in background mode on android? could you help?

**Comment by hutomosaktikartiko** [comment]
You can use Firebase Messaging https://pub.dev/packages/firebase_messaging to run code in the background with notification triggers

**Comment by markosole** [comment]
Yes you can, only one thing to consider if you want app to run in the background. Use service worker to keep app running in the background and in the same time you can use socket connection etc.
